tests/src/test/java/alluxio/client/fs/DirectAccessIntegrationTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.fs;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileInStream;
+import alluxio.client.file.FileOutStream;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.FileSystemUtils;
+import alluxio.conf.PropertyKey;
+import alluxio.grpc.CreateFilePOptions;
+import alluxio.testutils.LocalAlluxioClusterResource;
+
+import com.google.common.io.ByteStreams;
+import com.google.common.io.Files;
+import org.apache.commons.io.IOUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.File;
+
+/**
+ * Integration tests for path level configurations.
+ */
+public class DirectAccessIntegrationTest {
```

Comment:
Inherit from `BaseIntegrationTest` to get timeout and logging facilities.

```suggestion
public class DirectAccessIntegrationTest extends BaseIntegrationTest {
```